recipes/irida-uploader/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "irida-uploader" %}
+{% set version = "0.1" %}
+
+package:
+  name: '{{ name }}'
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/phac-nml/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 8080b411c0fcbc40d7792c65f1c9736186b0f5f436dc7115e0979cb80de1ea3e
+  patches: 
+    - upload_run.patch
+
+build:
+  number: 0
```

Comment:
noarch: python is correct

Reply:
Stating noarch: python caused the build to fail for me now, so I switched it to noarch:generic as per Devon's suggestion.